lochness/utils/source_check.py

```diff
@@ -216,23 +216,26 @@ def send_source_qc_summary(qc_fail_df: pd.DataFrame,
 
     send_detail(
         Lochness,
+        Lochness['sender'],
+        Lochness['file_check_notify'],
         'Files on source out of SOP',
         f'Daily updates {datetime.now(tz).date()}',
         'Dear team,<br><br>Please find the list of files on the source, which '
         'do not follow the SOP. Please move, rename or delete the files '
         'according to the SOP. Please do not hesitate to get back to us. '
-        '<br><br>Best wishes,<br>DPACC',
+        '<br><br>Best wishes,<br>DPACC<br><br><br>',
         table_str,
         lines,
-        yPlease let us know if any of the files above '
-        'should have passed QC')
+        'Please let us know if any of the files above should have passed QC'
+        )
```

Comment:
Not sure about the use of `should have`. Couldn't it be just `... above pass QC'` ?

Reply:
I'm asking people in the email to contact me back if the table includes any files that should have not been detected as "incorrect" in our algorithm.